ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -99,6 +99,12 @@ def __init__(self, instance):
         self._queue_tag_re = instance.get('queue_tag_re', {})
         self.queue_tag_re = self._compile_tag_re()
 
+        try:
+            mqcd_version = instance.get('mqcd_version', 6)
+            self.mqcd_version = getattr(pymqi.CMQC, 'MQCD_VERSION_{}'.format(int(mqcd_version)))
```

Comment:
```suggestion
            mqcd_version = int(instance.get('mqcd_version', 6))
            self.mqcd_version = getattr(pymqi.CMQC, 'MQCD_VERSION_{}'.format(mqcd_version))
```

(nit) That way `mqcd_version` is always `int`.

Reply:
If I do that then the `raise ConfigurationError("mqcd_version must be a number between 1 and 9. {} found.".format(mqcd_version))` would fail because `mqcd_version` is not defined.